src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User stepan;
+    private static RegistrationServiceImpl registerService;
+
+    @BeforeAll
+    static void init() {
+        registerService = new RegistrationServiceImpl();
+        stepan = new User();
+        stepan.setAge(18);
+        stepan.setId(12L);
+        stepan.setLogin("Stepan");
+        stepan.setPassword("test123");
+    }
+
+    @Test
+    void checkRegister_null_notOk() {
+        RuntimeException thrown = Assertions.assertThrows(RuntimeException.class, () -> {
+            registerService.register(null);
+        });
+        Assertions.assertEquals(thrown.getMessage(), "User cannot be null");
+    }
+
+    @Test
+    void validRegister_ok() {
+        User user = new User();
+        user.setAge(18);
+        user.setId(12L);
+        user.setLogin("User");
+        user.setPassword("test123");
+        Assertions.assertEquals(registerService.register(user), user);
+    }
+
+    @Test
+    void check_emptyLogin_notOk() {

```

Comment:
you are testing `register` method, so change `check` to `register`, the same for other methods